src/w2form.js

```diff
@@ -1264,7 +1264,6 @@
                 this.tmp_resize = function (event) { w2ui[obj.name].resize(); };
                 $(window).off('resize', 'body').on('resize', 'body', this.tmp_resize);
             }
-            setTimeout(function () { obj.resize(); obj.refresh(); }, 150); // need timer because resize is on timer
```

Comment:
Are you sure this needs to be removed? I fail to see how it relates to accept attribute